src/apm.js

```diff
@@ -17,23 +17,70 @@
  * under the License.
  */
 
-const { existsSync } = require('fs');
 const { join } = require('path');
-const { name, version } = require('../package.json');
+const { readFileSync } = require('fs');
```

Comment:
As this is dev only, what do you think about moving that file into `src/dev`?

Reply:
Sure thing.